python/pip.bzl

```diff
@@ -92,17 +42,11 @@ def pip_install(requirements = None, name = "pip", **kwargs):
         **kwargs (dict): Additional arguments to the [`pip_repository`](./pip_repository.md) repository rule.
     """
 
-    # Just in case our dependencies weren't already fetched
-    pip_install_dependencies()
-
-    pip_repository(
-        name = name,
-        requirements = requirements,
-        repo_prefix = "pypi__",
-        **kwargs
-    )
+    # buildifier: disable=print
+    print("pip_install is deprecated. Please switch to pip_parse. pip_install will be removed in a future release.")
+    pip_parse(requirements = requirements, name = name, **kwargs)
```

Comment:
Can you do something like `pip_parse(requirements_lock = requirements, ...` here so that you don't need to add a requirements parameter to `pip_parse`?

I'm not at all involved with `rules_python`, just excited about the feature!